### ISSUE TITLE
[Feature Request] Skip step/undo step/redo step

### ISSUE THREAD

**Issue by Ecselsior** [ffxivcode/AutoDuty#149]
Hello, would it be possible to have a button (and ideally a chat command) that allows you to skip to the next step (or go back a step). I am currently getting stuck in awkward situations such as my character attempting to pathfind to a previous platform. Being able to simply skip to the next step would mean I wouldn't have to stop autoduty and do the rest of the dungeon manually.

Here is an example in The Skydeep Cenote:
![image](https://github.com/user-attachments/assets/6e064c5d-0484-46b1-9a54-e57e14829fef)
![image](https://github.com/user-attachments/assets/9ee5ad58-9913-4a35-9466-c27798d30eaa)

**Comment by VeraNala** [comment]
You can *Stop* any path and click on the step you'd like to resume at in the *Main* tab, from there, hitting *Start* will resume it.

**Comment by Ecselsior** [comment]
> You can _Stop_ any path and click on the step you'd like to resume at in the _Main_ tab, from there, hitting _Start_ will resume it.

Hi, as you suggested I have clicked Stop and tried click on the next step, but now the button to start is greyed out, and I cannot click on any of the steps
![image](https://github.com/user-attachments/assets/124bb6d2-d4b9-463f-b091-032359332309)

The same issue is on the miniwindow.
![image](https://github.com/user-attachments/assets/c4dd28c0-5593-4a2f-af64-f96d9b138c84)

I was able to fix this by doing "Reset plugin data and reload", then downloading and reimporting the paths from github (which had disappeared), then randomly guessing as to which step I needed until I found the correct one, but obviously this is not a good solution.

Ideally, I would like to call to skip to next step from a macro, so that I would not have to manually interact with the addon window.
